lgc/patch/ConfigBuilderBase.cpp

```diff
@@ -239,6 +237,8 @@ void ConfigBuilderBase::setApiName(const char *value) {
 //
 // @param value : Value to set
 void ConfigBuilderBase::setPipelineType(Util::Abi::PipelineType value) {
+  if (value == Util::Abi::PipelineType::Invalid)
+    return;
```

Comment:
The `default` case below should be `llvm_unreachable`, right?

Reply:
setPipelineType should be align with PAL. I will check whether `isWholePipeline` is true to call the `setPipelineType` when we support unlinked shaders.